sickbeard/providers/anizb.py

```diff
@@ -32,8 +32,9 @@ def __init__(self):
         # Provider Init
         NZBProvider.__init__(self, "Anizb")
 
-        # Credentials
         self.public = True
+        self.supports_absolute_numbering = True
+        self.anime_only = True
         self.confirmed = True
```

Comment:
Honestly I don't know what self.confirmed is used for. It got copied from another prov.